kaitai_utils.py

```diff
@@ -97,10 +98,15 @@ def _object_to_dict(self, instance: Any, path: str) -> Generator[Dict[str, Any],
                 elif _is_list(value_object):
                     yield _to_lower_camel_case(key), self._list_to_dict(value_object, path)
                 elif isinstance(value_object, bytes):
-                    if len(value_object) < hansken_extraction_plugin.runtime.constants.MAX_CHUNK_SIZE:
```

Comment:
Might be personal preference:
There is a lot of indentation here and a cyclomatic complexity that makes it difficult to understand.
Can we split it into seperate methods somehow?
Maybe an easy first step could be to invert the check for `is_public_property` 
instead of checking if it is, check if it is not, and if it is not, just continue with the loop.
This removes 1 level of indentation for the rest of the `if-else` tree structure.